src/engine/SortPerformanceEstimator.cpp

```diff
@@ -32,14 +32,9 @@ IdTable createRandomIdTable(
   return result;
 }
 
-// TODO<C++20>: use std::is_sorted which then becomes constexpr.
 template <size_t I>
 constexpr bool isSorted(const std::array<size_t, I>& array) {
-  bool isSorted = true;
-  for (size_t i = 1; i < I; ++i) {
-    isSorted = isSorted && array[i] >= array[i - 1];
-  }
-  return isSorted;
+  return std::is_sorted(array.begin(), array.end());
```

Comment:
`libc++` does not yet support too much of the constexpr `<algorithm>` header.
I am planning to set up a `portability` library in the `util` space, that 
always makes dispatches of the form "if stdlib cupports  `feature`, then use `std::feature`, else use the following
(possibly simplified) implementation, provided by us. Then we can regularly check, which of those portability hacks is still needed. 
Do you want to start this library?
I suggest the folder `/util/portability` and then calling the files like the corresponding stdlib headers, so this would probably be in `/util/portability/algorithm.h`

Reply:
In this case libc++ uses `constexpr` since version 7 for `std::is_sorted`, so we're fine here.
https://github.com/llvm/llvm-project/blob/release/7.x/libcxx/include/algorithm#L3664-L3670

libstdc++ supports this as of version 10 of gcc:
https://github.com/gcc-mirror/gcc/blob/releases/gcc-10/libstdc++-v3/include/bits/stl_algo.h#L3225-L3229